src/components/Todo/Todo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import User from '../User';
+import './Todo.scss';
+
+const Todo = ({ title, completed, user }) => (
+  <>
+    <h1 className="tile">
+      {title}
+    </h1>
+    <p className={completed ? 'green' : 'red'}>
+      {String(completed)}
```

Comment:
no need to make a string of boolean
```suggestion
      {completed}
```